scripts/lib/utils

```diff
@@ -3,10 +3,8 @@
 
 readonly PROJECTS=(admiral cloud-prepare lighthouse shipyard subctl submariner submariner-charts submariner-operator)
 readonly REPO="quay.io/submariner"
-readonly ADMIRAL_CONSUMERS=(cloud-prepare lighthouse subctl submariner submariner-operator)
-readonly SHIPYARD_CONSUMERS=(admiral lighthouse subctl submariner submariner-operator)
-readonly OPERATOR_CONSUMES=(submariner lighthouse)
-readonly SUBCTL_CONSUMES=(submariner submariner-operator cloud-prepare lighthouse)
+readonly ADMIRAL_CONSUMERS=(cloud-prepare lighthouse submariner)
```

Comment:
Since it's now the same as `PROJECTS_PROJECTS` lets use that and be done with it.

You'll need to update the reference in `release.sh` as well.